src/flaskr/app/modules/resources/reset_token/schema.py

```diff
@@ -0,0 +1,6 @@
+from marshmallow import Schema, fields
+
+
+class ResetTokenSchema(Schema):
+    token = fields.String(required=True)
+    timestamp = fields.DateTime(required=True)

```

Comment:
I think you added some data to the database about your token but you didn't update the schema